tests/test_configurable_lookback_window.py

```diff
@@ -0,0 +1,85 @@
+"""Test tap configurable properties. Specifically the lookback_window"""
+import re
+import os
+from datetime import datetime as dt
+from datetime import timedelta
+
+from tap_tester import menagerie, connections, runner
+
+from base import BaseTapTest
+
+
+class ConversionWindowBaseTest(BaseTapTest):
+    """
+    Test tap's sync mode can execute with valid lookback_window values set.
+    Validate setting the lookback_window configurable property.
+    Test Cases:
+    Verify connection can be created, and tap can discover and sync with a lookback window
+    when passed in config else takes default value.
+    """
+    lookback_window = ''
+
+    def name(self):
+        return f"tt_stripe_lookback_window_{self.lookback_window}"
+
+    def get_properties(self):
+        """Configurable properties, with a switch to override the 'start_date' property"""
+        return_value = {
+            'start_date':dt.strftime(dt.utcnow() - timedelta(seconds=600), self.START_DATE_FORMAT),
```

Comment:
I don't think the 600 seconds is doing anything. The start date format should not include time infomation only date. So this should evaluate to something like `05-25-22T00:00:00Z`.

Reply:
The `START_DATE_FORMAT` was already defined and used in the `test_start_date.py`, hence we've utilized the same. Moreover, the `timedelta(600)` is just to create spare time between the start date and the sync time, hence we can get lookback.

Reply:
I see, but  subtracting 10 minutest from today's date is unlikely to actually change the value that you set as start_date. It's going to be utc midnight of today. Applying the start date format https://github.com/singer-io/tap-stripe/blob/cbdd805335adc6031250cad6719ed2f2c82e8be7/tests/base.py#L29 will truncate the time information from the datetime value. Not a big deal, but just so you are aware.

Reply:
Ohh understood what you meant, the time information would be made 0. Thanks for explaining.

Reply:
Lets remove `timedelta(seconds=600)` if it doesn't have any significance.